src/app/components/trips/trips.component.html

```diff
@@ -1,11 +1,12 @@
 <div class="size-screen flex flex-col items-center justify-center gap-4 bg-white">
 <h1 class="text-3xl py-8 font-bold w-5/6">My Trips</h1>
-    @for (trip of trips; track trip.id) {
-        <app-trip-card [name]="trip.name" [startDate]="trip.startDate" [endDate]="trip.endDate" [location]="trip.location"></app-trip-card>
+@for (trip of selectedTrips$ | async; track trip.id){
+    <a [routerLink]="['/trip', trip.id]" (click)="setTrip(trip.id ?? '')">
+        <app-trip-card [trip]="trip"></app-trip-card>
+    </a>
     } @empty {
         <h2 class="text-2xl">Ready for an adventure? Create your first Trip!</h2>
```

Comment:
I think trip here should be `trip` as its not a proper noun

Reply:
I do not understand. What is the difference?